src/components/Loader/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { Link } from 'react-router-dom';
+import { Person } from '../../../types';
+
+interface Props {
+  person: Person,
+}
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  return (
+    <Link
+      to={`../${person.slug}`}
+      className={classNames({
+        'has-text-danger': person.sex === 'f',
+      })}
+    >
+      {person?.name}
+    </Link>
+  );
+};
```

Comment:
It's better to destructure `person` 

` const { name, sex, slug } = person;`

Reply:
Good point